.travis.yml

```diff
@@ -1,11 +1,13 @@
-language: ruby
+---
 sudo: false
-
-script: 'bundle exec rake validate lint spec SPEC_OPTS="--format documentation"'
-
+language: ruby
+bundler_args: --without system_tests
+script: "bundle exec rake validate lint spec SPEC_OPTS='--format documentation'"
 matrix:
   fast_finish: true
-  includes:
+  include:
+  - rvm: 1.8.7
```

Comment:
Is this ruby 1.8.7 node still desired? I've been removing them from other modules.